boards/nrf52dk/Makefile.include

```diff
@@ -14,7 +14,7 @@ export JLINK_DEVICE := nrf52
 
 # special options when using SoftDevice
 ifneq (,$(filter nordic_softdevice_ble,$(USEPKG)))
-export JLINK_PRE_FLASH := loadfile $(BINDIR)/softdevice.hex
+export JLINK_PRE_FLASH := erase\nloadfile $(BINDIR)/softdevice.hex
```

Comment:
I confirmed with a quick test that this is related to bash versus dash.
in bash, echo "erase\nloadfile" will output erase\nloadfile. You will need to add -e to have the \n handled: echo -e "erase\nloadfile" will output:
erase
loadfile

Using dash, echo -e is not supported and the escape characters are handled.
echo "erase\nloadfile" will output:
erase
loadfile
While echo -e "erase\nloadfile" will output:
-e erase
loadfile
It is recommended to use printf for portability as per : https://wiki.ubuntu.com/DashAsBinSh#echo
This will mean: replace echo "${JLINK_PRE_FLASH}" >> pathtobin/burn.seg
by
printf "${JLINK_PRE_FLASH}\n"  >> pathtobin/burn.seg

Let me know what you think